themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -556,8 +556,13 @@ module.controller('RealmLocalizationCtrl', function($scope, Current, $location,
 
     $scope.updateRealmSpecificLocalizationTexts = function() {
         RealmSpecificLocalizationTexts.get({id: realm.realm, locale: $scope.selectedRealmSpecificLocales }, function (updated) {
-            $scope.localizationTexts = updated;
+            $scope.localizationTexts = sortObjectByKeys(updated);
         })
+
+    }
```

Comment:
A surplus blank line slipped in here, please remove it.

```suggestion
        })
    }
```

Reply:
@ahus1 thanks for the review. Sorry this new line was indeed unintended. I updated the commit accordingly.